v20.2/admin-ui-sessions-page.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Sessions Page
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+On a secure cluster, this area of the Admin UI can only be accessed by a user with [`VIEWACTIVITY`](authorization.html#assign-privileges) privileges. Note that non-`admin` users will see only their own sessions, while `admin` users see sessions for all users.
+{{site.data.alerts.end}}
+
+The **Sessions** page of the Admin UI provides details of all open sessions on the cluster.
+
+To view this page, [access the Admin UI](admin-ui-overview.html#admin-ui-access) and click **Sessions** in the left-hand navigation.
+
+## Sessions list
+
+Use the **Sessions** list to see the open sessions on the cluster. This includes active and idle sessions. 
+
+{{site.data.alerts.callout_info}}
+An session is *active* if it has an open transaction, and *idle* if it has no open transaction. Active sessions consume [hardware resources](recommended-production-settings.html#hardware).
```

Comment:
It might be worth it to clarify that this is true even if the session is running a single statement outside of the context of an explicit transaction. We call this an implicit transaction, but I'm not sure if this is going to be completely clear to a user.